doc/source/tutorial/stats/sampling_tdr.rst

```diff
@@ -98,8 +98,43 @@ to see how well the generator fits the given distribution. These are:
     >>> rng.squeeze_hat_ratio == rng.squeeze_area / rng.hat_area
     True
 
+The distribution can be truncated by passing a domain parameter:
+
+    >>> urng = np.random.default_rng()
+    >>> rng = TransformedDensityRejection(dist, domain=[0, 1], numpy_rng=urng)
+    >>> rng.rvs(10)
+    array([0.05452512, 0.97251362, 0.49955877, 0.82789729, 0.33048885,
+           0.55558548, 0.23168323, 0.13423275, 0.73176575, 0.35739799])
+
+If the domain is not specified, the ``support`` method of the ``dist`` object
+is used to determine the domain. If the ``dist`` object does not provide a
+``support`` method, the support is assumed to be ``[-np.inf, np.inf]``.
+
+    >>> class StandardNormal:
+    ...     def pdf(self, x):
```

Comment:
I think one should use domain to truncate in this case. The support of standard normal should be the real line. This might be confusing

Reply:
> I think one should use domain to truncate in this case. The support of standard normal should be the real line. This might be confusing

Makes sense. I will change it this way.

Reply:
Done.